ide/che-core-ide-templates/src/main/resources/samples.json

```diff
@@ -1184,5 +1184,40 @@
     "tags": [
       "SDK"
     ]
+  },
+  {
+    "name": "python-hello-world",
+    "displayName": "python-hello-world",
+    "path": "/python-hello-world",
+    "description": "Python Hello World Project.",
+    "projectType": "python",
+    "mixins": [],
+    "attributes": {
+      "language": [
+        "python"
+      ]
+    },
+    "modules": [],
+    "problems": [],
+    "source": {
+      "type": "git",
+      "location": "https://github.com/tolusha/python-hello-world.git",
+      "parameters": {}
+    },
+    "commands": [
+      {
+        "name": "run",
+        "type": "custom",
+        "commandLine": "cd ${current.project.path} && python hello-world.py",
```

Comment:
I may be wrong but `${current.project.path}` is not resolved in Theia, hence the command fails. And anyway to be able to resolve it a user is forced to select the project before running the command.

That said I think the a better way to `cd` is using the project path (since you know it):

```
cd ${CHE_PROJECTS_ROOT}/python-hello-world
```